MantidPlot/src/PythonScripting.cpp

```diff
@@ -222,12 +222,20 @@ void PythonScripting::shutdown() {
 
 void PythonScripting::setupPythonPath() {
   using Mantid::Kernel::ConfigService;
-// The python sys.path is updated as follows:
-//   - the current working directory is inserted as position 0 to mimic the
-//     behaviour of the vanilla python interpreter
-//   - the directory of MantidPlot is added after this to find our bundled
-//   - modules
-//
+  // First add the directory of the executable as a sitedir to process
+  // any .pth files
+  const auto appPath = ConfigService::Instance().getPropertiesDir();
+  PyObject *sitemod = PyImport_ImportModule("site");
+  Py_DECREF(PyObject_CallMethod(sitemod, "addsitedir", STR_LITERAL("(s)"),
```

Comment:
I think doing `STR_LITERAL("addsitedir")` instead of just `"addsitedir"` should eliminate the compiler warning,